projects/1.18-fabric/assets/toms-storage-fabric/toms_storage/lang/zh_cn.json

```
@@ -0,0 +1,102 @@
+	"tooltip.toms_storage.filtered": "可过滤",
+	"tooltip.toms_storage.trim": "将容器与存储驱动器连接。",
+	"tooltip.toms_storage.crafting_terminal": "你可以在其中拿取你的物品。\\并可以将物品存储在存储网络之中。\\合成终端中自带合成台。",
+	"tooltip.toms_storage.storage_terminal": "你可以在其中拿取你的物品。\\并可以将物品存储在存储网络之中。",
+	"tooltip.toms_storage.wireless_terminal": "扩展你的终端访问范围。\\访问范围：%d个方块",
+	"tooltip.toms_storage.paint_kit": "喷涂方块使他们\\贴近于周围环境。",
+	"tooltip.toms_storage.open_crate": "将其面前方块\\转化为一个存储空间\\警告: 这将创建物品实体。",
+	"tooltip.toms_storage.inventory_connector": "连接其周围所有的容器\\使用存储终端访问你所存储的所有物品\\存储桥连方块可以填补空隙。",
+	"tooltip.toms_storage.inventory_proxy": "可以拓展其指针朝向的方块的存储交互面。\\使用钻石在其侧面启用过滤面。\\放置容器在其交互面，然后在物品槽中放入要过滤的物品。\\空物品槽允许所有物品放入。放入物品的数量将决定被过滤的最大堆叠数。\\若想使用其他设置请重命名过滤面容器。\\格式为：'<key>=[value],[key2]=[value2],...'\\选项:",
+	"tooltip.toms_storage.inventory_proxy.key": " - '%s' %s",
+	"tooltip.toms_storage.inventory_proxy.value": " - '%s=%s' %s",
+	"tooltip.toms_storage.inventory_proxy.ignoreSize": "过滤时忽略物品堆叠数量",
+	"tooltip.toms_storage.inventory_proxy.maxCount.arg": "<max value>",
+	"tooltip.toms_storage.inventory_proxy.maxCount.desc": "设置空物品栏的最大堆叠数",
+	"tooltip.toms_storage.inventory_cable": "跨长距离的连接容器\\存储线缆连接器\\必须接入容器\\存储网络中必须存在着\\一个存储驱动器。",
+	"tooltip.toms_storage.inventory_hopper": "将物品从存储网络中导入或者导出\\拿着一个物品右键漏斗以设置该物品为过滤物品。\\空手Shift+右键以清除过滤器。",
+	"tooltip.toms_storage.loop": "检测到你的存储网络中发生了循环",
+	"tooltip.toms_storage.amount": "物品数量: %d",
+	"tooltip.toms_storage.sorting_0": "按物品数量进行排序",
+	"tooltip.toms_storage.sorting_1": "按物品名称进行排序",
+	"tooltip.toms_storage.search_0": "标准搜索",
+	"tooltip.toms_storage.search_1": "自动搜索",
+	"tooltip.toms_storage.search_2": "保持标准搜索",
+	"tooltip.toms_storage.search_3": "保持自动搜索",
+	"tooltip.toms_storage.search_4": "%s同步标准搜索",
+	"tooltip.toms_storage.search_5": "%s同步自动搜索",
+	"tooltip.toms_storage.search_6": "保持%s同步标准搜索",
+	"tooltip.toms_storage.search_7": "保持%s同步自动搜索",
+	"tooltip.toms_storage.ctrlMode_0": "应用能源控制偏好\\拿取单个物品：Shift+右击\\移动单个物品：Ctrl+Shift+右击",
+	"tooltip.toms_storage.ctrlMode_1": "精致存储控制偏好\\拿取单个物品：中键点击\\移动单个物品：Shift+中键点击",
+	"tooltip.toms_storage.ctrlMode_2": "原版控制偏好\\拿单个物品：右键单击\\移动单个物品：Shift+右击",
+	"tooltip.toms_storage.pull_0": "合成时不从玩家背包中提取物品",
+	"tooltip.toms_storage.pull_1": "合成时从玩家背包中提取缺失的物品",
+	"tooltip.toms_storage.filter_item": "过滤物: %s",
+	"tooltip.toms_storage.empty": "空",
+	"tooltip.toms_storage.level_emitter": "根据存储系统内的物品发出红石信号",
+	"tooltip.toms_storage.lvlEm_lt_0": "在存储网络中物品超过指定值时发出红石信号",
+	"tooltip.toms_storage.lvlEm_lt_1": "在存储网络中物品少于指定值时发出红石信号",
+	"tooltip.toms_storage.adv_wireless_terminal": "按住Shift键并对终端单击鼠标右键将其绑定\\范围：%d个方块\\在8个方块内建立%d级信标可在绑好的终端有效半径内从同一维度的任何位置访问。\\在8个方块内建立%d级信标可在绑好的终端有效半径内从同一维度的任何位置访问。\\必须加载终端所在的区块",
+	"tooltip.toms_storage.adv_wireless_terminal.bound": "绑定到坐标(%s, %s, %s)于%s",
+	"tooltip.toms_storage.link_public_0": "私有频道",
+	"tooltip.toms_storage.link_public_1": "公共频道",
+	"tooltip.toms_storage.link_remote_0": "访问远程容器",
+	"tooltip.toms_storage.link_remote_1": "提供远程访问的容器",
+	"tooltip.toms_storage.items_missing": "某些物品可能无法放入合成栏",

```

| **审查级别**    | 备注 |
|--------------|----|
| ⚪ 轻微 / 不阻止合并但需三思  |    |
```suggestion
	"tooltip.toms_storage.items_missing": "某些物品可能无法放入合成方格",
```
原文：**Some items may not be inserted into the crafting grid**